features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -854,6 +911,90 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     };
 
+    const hybridFlowResponseTypeMap: Map<string, string> = new Map([
+        [
+            t("applications:forms.inboundOIDC.sections.hybridFlow.hybridFlowResponseType." +
+            "fields.children.code_token.label"),
+            CODE_TOKEN
+        ],
+        [
+            t("applications:forms.inboundOIDC.sections.hybridFlow.hybridFlowResponseType.fields." +
+            "children.code_idtoken.label"),
+            CODE_IDTOKEN
+        ],
+        [
+            t("applications:forms.inboundOIDC.sections.hybridFlow.hybridFlowResponseType.fields." +
+            "children.code_idtoken_token.label"),
+            CODE_IDTOKEN_TOKEN
+        ]
+    ]);
+
+    const getHybridFlowResponseTypesHintDescription = (value: string): string => {
+        switch (value) {
+            case ApplicationManagementConstants.CODE_TOKEN:
+                return t("applications:forms.inboundOIDC.fields.hybridFlow.hybridFlowResponseType.children." +
+                    "code_token.hint");
+            case ApplicationManagementConstants.CODE_IDTOKEN_TOKEN:
+                return t("applications:forms.inboundOIDC.fields.hybridFlow.hybridFlowResponseType.children." +
+                    "code_idtoken_token.hint");
+            default:
+                return null;
+        }
+    };
+
+    const getHybridFlowResponseTypes = () => {
+        type CheckboxChildWithIndex = CheckboxChild & { index?: number; };
+        const allowedList: CheckboxChildWithIndex[] = [];
+
+        hybridFlowResponseTypeMap.forEach((responseType: string, label: string) => {
+
+            const responseTypeList: CheckboxChildWithIndex = {
+                label: modifyResponseTypeLabel(responseType, label),
+                value: responseType
+            };
+
+            const description: string = getHybridFlowResponseTypesHintDescription(responseType);
+
+            if (description && !isM2MApplication) {
+                responseTypeList.hint = {
+                    content: description,
+                    header: ""
+                };
+            }
+
+            allowedList.push(responseTypeList);
+            // responseTypes.push({ label: responseTypeLabel, value });
+        });
+
+        return allowedList;
+    };
+
+    const modifyResponseTypeLabel = (value: string, label: string) => {
```

Comment:
Let's add doc comments to these functions

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6271/commits/11295ba4d8273091b1934290f09ab0ab4d02ac0d